src/applications/facility-locator/components/search-results-items/UrgentCareResult.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LocationAddress from './common/LocationAddress';
+import LocationDirectionsLink from './common/LocationDirectionsLink';
+import LocationPhoneLink from './common/LocationPhoneLink';
+
+import { CLINIC_URGENTCARE_SERVICE, LocationType } from '../../constants';
+import recordEvent from 'platform/monitoring/record-event';
+
+/**
+ * Urgent care
+ * - All urgent care
+ * - VA urgent care
+ * - Community urgent care providers (in VA's network)
+ *
+ * Community providers (in VA’s network)
+ *  - Clinic/Center - Urgent Care
+ */
+const UrgentCareResult = ({ provider, query }) => {
+  const { name } = provider.attributes;
+  const distance = provider.distance;
+
+  const urgentCareCall = (prov, resultQuery) => {
+    const content = () => (
+      <p>
+        {' '}
+        Before going to a clinic for urgent care, please call the facility to
+        confirm that it's open and able to provide the care you need.{' '}
+      </p>
+    );
+
+    if (
+      resultQuery.facilityType === LocationType.URGENT_CARE &&
+      resultQuery.serviceType === 'NonVAUrgentCare'
+    ) {
+      return content();
+    }
+    if (
+      resultQuery.facilityType === LocationType.CC_PROVIDER &&
+      resultQuery.serviceType === CLINIC_URGENTCARE_SERVICE
+    ) {
+      return content();
+    }
+
+    if (
+      prov.type === LocationType.CC_PROVIDER &&
+      (!resultQuery.serviceType || resultQuery.serviceType === 'AllUrgentCare')
+    ) {
+      return <p>Call to confirm services and hours</p>;
+    }
+    return null;
```

Comment:
Can we add a line break above this line? I think it helps w/readability, otherwise it looks like this line is grouped with the previous block.